pkg/operator/ceph/pool/controller.go

```diff
@@ -189,6 +190,17 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 
 	updateStatus(r.client, request.NamespacedName, k8sutil.ReconcilingStatus)
 
+	// If the CephCluster has enabled the "pg_autoscaler" module
+	// we will turn enforce the autoscaling mode to one
+	//
+	// This is only valid for Nautilus cluster
```

Comment:
Why is this only valid for nautilus? I don't see any nautilus checks.

Reply:
The only check is the presence of the "pg_autoscaler" module in the CephCluster spec which we only recommend on Nautilus clusters and not on Octopus. I'll read the label to get the right version instead.